pkg/agent/storage/keyvalue_store_bbolt_test.go

```diff
@@ -93,8 +92,8 @@ func Test_Updates(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Parallel()
 
-			db := createDb(t)
-			bc := NewBucketConsumer(log.NewNopLogger(), db, tt.name)
```

Comment:
I'm going to refactor this test so it's agnostic to the key value store type. Then it can be run against each concrete key-value store implementation, since they all should behave the same from a client's perspective.